index.html

```diff
@@ -260,11 +260,12 @@ <h1>Contact me</h1>
                 I'd love to help with it </p>
```

Comment:
- [ ] On line 257, Please change `<h1>` heading with other `<hx>` headings like h2, h3, h4..... A webpage should only have one `<h1>` heading and you are already using it in your hero section. Read more [here](https://www.investisdigital.com/blog/seo/using-h1-tag-improves-search-engine-ranking#:~:text=Your%20website%20should%20have%20only,to%20an%20h2%20or%20h3.)